### PR TITLE
chore(generators): fix new component templates

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__ [hunk 1]
```diff
@@ -2,17 +2,14 @@ import { html } from '@microsoft/fast-element';
...
-	'control',
-	[`connotation-${connotation}`, Boolean(connotation)],
-	[`appearance-${appearance}`, Boolean(appearance)],
-	[`shape-${shape}`, Boolean(shape)],
-	[`size-${size}`, Boolean(size)],
-);
+}: <%= className %>) =>
+	classNames(
+		'control',
+	);
```
Comment (YonatanKra) [inline tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__:12]: This will fail because the class would not have `connotation` etc.

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__ [hunk 2]
```diff
@@ -2,17 +2,14 @@ import { html } from '@microsoft/fast-element';
 import type { ViewTemplate } from '@microsoft/fast-element';
 import type { ElementDefinitionContext, FoundationElementDefinition } from '@microsoft/fast-foundation';
 import { classNames } from '@microsoft/fast-web-utilities';
-import type { <%= className %> } from './<%= fileName %>.base';
+import type { <%= className %> } from './<%= fileName %>';
 
 const getClasses = ({
 	connotation, appearance, shape, size, iconTrailing,
```
Comment (YonatanKra) [inline tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__:8]:
```suggestion
	text,
```

File: tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__ [hunk 3]
```diff
@@ -2,17 +2,12 @@ import { html } from '@microsoft/fast-element';
...
-const getClasses = ({
-	connotation, appearance, shape, size, iconTrailing,
-}: Badge) => classNames(
-	'control',
-	[`connotation-${connotation}`, Boolean(connotation)],
-	[`appearance-${appearance}`, Boolean(appearance)],
-	[`shape-${shape}`, Boolean(shape)],
-	[`size-${size}`, Boolean(size)],
-);
+const getClasses = (_: <%= className %>) =>
```
Comment (YonatanKra) [inline tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__:7]: Are we sure we want to go this way? I mean - `_` is meaningless and doesn't give the user a glance at what it is. Maybe `elementProperties`?

Comment (yinonov) [reply tools/generators/vivid-component/files/__fileName__.template.ts__tmpl__:7]: the getClasses approach isn't mandatory. dont mind removing it all-together

**Pull request merged**